seismicpro/src/seismic_index.py

```diff
@@ -183,6 +184,37 @@ def merge(self, x, **kwargs):
         indices = df.index.unique().sort_values()
         return type(self).from_index(index=indices, idf=df, index_name=self.name)
 
+    def concat(self, other):
+        """ Concatenate vertically current and `other` indices DataFrames.
+
+        Parameters
+        ----------
+        other : Index or sequence of Index
+            Indices which dataframes are being concatenated.
+
+        Returns
+        -------
+            : TraceIndex
+            Concatenated index.
+        """
+        warn_msg = 'Concatenation of several `Index` instances may result in non-unique index.'
+        warnings.warn(warn_msg, RuntimeWarning)
+
+        if not isinstance(other, (list, tuple)):
+            other = [other]
+
+        df = [self.get_df()]
+        other_dfs = [index.get_df() for index in other]
+        concat_dfs = pd.concat(df + other_dfs, ignore_index=True)
```

Comment:
imo, this is more clear, because `df` is usually a dataframe, not a list of dataframes
```suggestion
        concat_dfs = pd.concat([self.get_df()] + other_dfs, ignore_index=True)
```